src/Umbraco.Web.UI.Client/examples/entity-content-type-condition/workspace-view-unique.element.ts

```diff
@@ -0,0 +1,20 @@
+import { html, customElement } from '@umbraco-cms/backoffice/external/lit';
+import { UmbLitElement } from '@umbraco-cms/backoffice/lit-element';
+
+@customElement('umb-example-entity-content-type-condition-unique')
```

Comment:
I think we should move the 'condition' part of the element name to the end. It will make it more clear that the condition is about the "content-type-unique". Remember to change it at the bottom of the file too.

```suggestion
@customElement('umb-example-entity-content-type-unique-condition')
```